rskj-core/src/main/java/co/rsk/cli/RskCli.java

```diff
@@ -76,6 +77,12 @@ static class NetworkFlags {
     @CommandLine.Option(names = {"-X"}, description = "Read arguments in command line")
     private List<String> xArguments;
 
+    @CommandLine.Option(names = {"--sync-mode"}, description = "Set Synchronization mode. Valid options are <full | snap>")
+    private String syncMode;
```

Comment:
I'd suggest to make it strongly typed, eg. with help enum:
```
enum SyncMode { full, snap }
```

Reply:
solved, thanks for the suggestion!